app/assets/stylesheets/style.scss

```diff
@@ -10,6 +10,7 @@ html {
   line-height: 1.15;
   -webkit-text-size-adjust: 100%;
   -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
+  overflow: hidden;
```

Comment:
Before I couldn't use this in our css body because we had the search after our creepy hands in the welcome page but now that the search has been moved, I moved it in here to prevent that annoying white space from bouncing around when scrolling